corehq/apps/cloudcare/templates/formplayer/case_list.html

```diff
@@ -17,48 +17,56 @@
-          <% } %>
-          <% _.each(headers, function(header, index) { %>
-          <% if (columnVisible(index)) { %>
-          <% if (columnSortable(index)) { %>
-          <th class="module-case-list-header header-clickable formplayer-request" data-id="<%- index %>"> <%- header %></th>
-          <% } else { %>
-          <th class="module-case-list-header" data-id="<%- index %>"> <%- header %></th>
-          <% } %>
-          <% } %>
-          <% }); %>
-        </tr>
-        </thead>
-        <% } %>
-        <% if (hasNoItems) { %>
-        <tbody>
-        <tr>
-          <td class="module-case-list-column module-case-list-column-empty" colspan="<%-headers.length%>">
-            <% if (noItemsText === null) {%>
-              <div class="alert alert-info">{% trans "List is empty." %}</div>
-            <% } else { %>
-              <div class="alert alert-info"><%- noItemsText %></div>
+    <div style="<%- columnStyle %>">
+      <div>
+        <% if (useGrid) { %>
+            <section  class="js-case-container clearfix list-cell-container-style"></section>
+          </div>
+        <% } else { %>
+        <div class="module-case-list-table-container">
+          <table class="table module-table module-table-case-list">
+<% if(!useTiles) { %>
+            <thead>
+            <tr>
+              <% if (isMultiSelect && !hasNoItems) { %>
+              <th class="module-case-list-header"><input type="checkbox" id="select-all-checkbox"/></th>
+              <% } %>
+              <%  _.each(headers, function(header, index) { %>
+              <% if (columnVisible(index)) { %>
+              <% if (columnSortable(index)) { %>
+              <th class="module-case-list-header header-clickable formplayer-request" data-id="<%- index %>"> <%- header %></th>
+              <% } else { %>
+              <th class="module-case-list-header" data-id="<%- index %>"> <%- header %></th>
+
+              <% } %>
+              <% } %>
+              <% }); %>
+            </tr>
+            </thead>
+<% } %>
+            <% if (hasNoItems) { %>
+            <tbody>
+            <tr>
+              <td class="module-case-list-column module-case-list-column-empty" colspan="<%-headers.length%>">
+                <% if (noItemsText === null) { %>
+                  <div class="alert alert-info">{% trans "List is empty." %}</div>
+                <% } else { %>
+                  <div class="alert alert-info"><%- noItemsText %></div>
+                <% } %>
+              </td>
+            </tr>
+            </tbody>
             <% } %>
-          </td>
-        </tr>
-        </tbody>
+            <tbody class="wrapper js-case-container">
+            </tbody>
+          </table>
+        </div>
         <% } %>
-        <tbody class="wrapper js-case-container">
-        </tbody>
-      </table>
+      </div>
+      <% if (showMap && !hasNoItems) { %>
+        <div id="module-case-list-map" class="map" style="height: 100%"></div>
```

Comment:
Can you move the style into a stylesheet? There are a bunch in `corehq/apps/hqwebapp/static/cloudcare/less/formplayer-common`